pkg/router/router_test.go

```diff
@@ -445,7 +559,12 @@ func TestConfigTemplate(t *testing.T) {
 		for _, expectation := range expectations {
 			t.Run(name, func(t *testing.T) {
 				if err := expectation.Match(parser); err != nil {
-					t.Fatalf(err.Error())
+					if content, err := ioutil.ReadFile(config); err != nil {
+						t.Error(err)
+					} else {
+						t.Log("haproxy.config:", string(content))
+					}
```

Comment:
We probably don't need to log the content when there is no error.

Reply:
Is this for debugging?  Please remove.

Reply:
I asked Miheer to add this.  It is only printed when the test fails, and it is helpful for diagnosing the cause of the failure (at least it was useful to me).

Reply:
I also find it useful